examples/hpc/dla-driver/src/lib.rs

```diff
@@ -134,7 +164,7 @@ impl MemoryBank {
             MemoryBank::Bank15 => MEMORY_BANK_15_OFFSET,
         }
     }
-    fn value(&self) -> usize {
+    fn from(&self) -> usize {
```

Comment:
A function like this should be called `into_usize` if implemented in `MemoryBank`. However, these conversions are usually implemented using `From`, `TryFrom`, `Into` and `TryInto`.

`From` and `TryFrom` are usually preferred over `*Into` because they provide the respective `Into` and `TryInto` impls automatically. Embedded code is an exception to this rule though, as people tend to be careful with code size. Another exception which applies here is the Rust orphan rule, i.e., if the target type is a foreign/primitive type, preventing `impl From<_> for <foreign>`.

**I think in this case the most appropriate choice is to implement `impl Into<usize> for MemoryBank {}`**.

Reply:
`impl Into<usize> for MemoryBank {}` implemented and used